### PR TITLE
Re-target to netstandard2.0

### PR DESCRIPTION
**Pull request by netspark**
#### Why?
We need to have the library on dotnetcore

#### How?
Simple replace of the target framework and removal of the nuspec file

### CODE REVIEW

File: src/Intercom/Intercom.csproj [hunk 1]
```diff
@@ -1,24 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
-    <ReleaseVersion>3.0.0</ReleaseVersion>
-    <PackageId>Intercom.Dotnet.Client</PackageId>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
```
Comment (seankearney) [inline src/Intercom/Intercom.csproj]: We've seen issues when targeting a single framework with the plural `TargetFrameworks`. I believe one targeting a single framework it should be `<TargetFramework>` (singular version).

File: src/Intercom/Intercom.csproj [hunk 2]
```diff
@@ -1,24 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
-    <ReleaseVersion>3.0.0</ReleaseVersion>
-    <PackageId>Intercom.Dotnet.Client</PackageId>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <ReleaseVersion>3.1.0</ReleaseVersion>
+    <PackageId>Mdc.Intercom.DotNetCore</PackageId>
```
Comment (seankearney) [inline src/Intercom/Intercom.csproj]: Let's leave the package name as-is. We can identify our custom fork with a version identifier

File: src/Intercom/Intercom.csproj [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <ReleaseVersion>3.1.0</ReleaseVersion>
+    <PackageId>Mdc.Intercom.DotNetCore</PackageId>
     <PackageIconUrl>https://raw.githubusercontent.com/intercom/intercom-dotnet/master/src/assets/Intercom.png</PackageIconUrl>
     <Owners>Intercom</Owners>
     <PackageProjectUrl>https://github.com/intercom/intercom-dotnet</PackageProjectUrl>
-    <Summary>The official Intercom API client library for .NET</Summary>
-    <Title>Intercom Dotnet Client</Title>
-    <Description>The official Intercom API client library for .NET</Description>
+    <Summary>This is the full copy of the official Intercom API client library for .NET but published into NuGet feed</Summary>
```
Comment (seankearney) [inline src/Intercom/Intercom.csproj]: `The digitaldjpool fork of the Intercom API client library for .NET Core`

File: src/Intercom/Intercom.csproj [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
+    <Title>Intercom Dotnet Client with dotnetcore support</Title>
+    <Description>The fork of the official Intercom API client library for .NET with net standard support published into NuGet feed</Description>
```
Comment (seankearney) [inline src/Intercom/Intercom.csproj]: `The fork of the official Intercom API client library for .NET Standard 2.0`

**Comment by netspark** [comment]
Here is what I get, if I change `TargetFrameworks` -> `TargetFramework`
image

File: src/Intercom/Intercom.csproj [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
     <PackOnBuild>true</PackOnBuild>
+    <Version>3.1.0</Version>
```
Comment (seankearney) [inline src/Intercom/Intercom.csproj]: Change this to `3.1.0-djp-01` which will indicate that this is a DJ Pool "pre-release" version of 3.1.0.

**Review by seankearney — CHANGES_REQUESTED** [review]

**Comment by seankearney** [comment]
> Here is what I get, if I change `TargetFrameworks` -> `TargetFramework`
> image

That middle error is likely a result because of the trailing `;` Use `<TargetFramework>netstandard2.0</TargetFramework>`

File: src/Intercom/Intercom.csproj (hunk 5 above)
Comment (seankearney) [reply src/Intercom/Intercom.csproj]: 2.1.2-djp-01

**Comment by netspark** [comment]
> 
> 
> > Here is what I get, if I change `TargetFrameworks` -> `TargetFramework`
> > image
> 
> That middle error is likely a result because of the trailing `;` Use `<TargetFramework>netstandard2.0</TargetFramework>`

This still gives me build errors even after stripping the `;`. It gives no errors for 
 `<TargetFrameworks>netstandard2.0</TargetFrameworks>`
@seankearney

**Pull request merged**